src/sentry/utils/suspect_resolutions_releases/get_suspect_resolutions_releases.py

```diff
@@ -0,0 +1,51 @@
+from typing import Sequence
+
+from sentry import features
+from sentry.models import Group, GroupRelease, GroupStatus, Project, Release, ReleaseProject
+from sentry.signals import release_created
+from sentry.tasks.base import instrumented_task
+from sentry.utils.suspect_resolutions_releases import ALGO_VERSION, analytics
+
+
+@release_created.connect(weak=False)
+def record_suspect_resolutions_releases(release, project, **kwargs) -> None:
+    if features.has("projects:suspect-resolutions-releases", project):
+        get_suspect_resolutions_releases.delay(release, project)
```

Comment:
I think now we have to make sure we're adding a 1 hour delay here to see the effects of the release, right? Otherwise, when the release is just created, we won't have any usable error rate data cause the release was just created.

Reply:
Yeah, not sure what the best way to go about adding a 1 hour delay here is.